charts/argo/templates/submit-workflow-template-role.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.workflow.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
Can we not include this role as part of this PR? I am not sure it is relevant to the 2.11 upgrades.

Reply:
will remove - it's also related to workflows (see https://github.com/argoproj/argo/pull/3488).